$NetBSD: patch-CMakeLists.txt,v 1.2 2012/10/03 14:55:52 othyro Exp $

Install config file to ${EGDIR}, then let ${CONF_FILES} handle it.

--- CMakeLists.txt.orig	2012-09-07 19:09:16.000000000 +0000
+++ CMakeLists.txt
@@ -427,7 +427,7 @@ IF( ENABLE_SDL )
 
     INSTALL(PROGRAMS ${CMAKE_CURRENT_BINARY_DIR}/vbam DESTINATION bin)
     INSTALL(FILES ${CMAKE_CURRENT_SOURCE_DIR}/src/sdl/vbam.cfg-example
-            DESTINATION ${SYSCONFDIR}
+            DESTINATION ${CMAKE_INSTALL_PREFIX}/share/examples/vbam
             RENAME vbam.cfg)
     INSTALL(FILES ${CMAKE_CURRENT_SOURCE_DIR}/src/vba-over.ini DESTINATION ${DATA_INSTALL_DIR})
 ENDIF( ENABLE_SDL )
